static_src/actions/user_actions.js

```diff
@@ -212,30 +218,47 @@ const userActions = {
   },
 
   fetchCurrentUserRole(userGuid) {
+    let entityGuid;
+
+    const currentViewType = UserStore.currentlyViewedType;
+    const currentSpaceGuid = SpaceStore.currentSpaceGuid;
+    const currentOrgGuid = OrgStore.currentOrgGuid;
+
     if (!userGuid) {
       return Promise.reject(new Error('guid is required'));
     }
 
-    const orgGuid = OrgStore.currentOrgGuid;
+    if (currentViewType == SPACE_NAME){
+      entityGuid = currentSpaceGuid;
+    } else {
+      entityGuid = currentOrgGuid;
+    }
 
     AppDispatcher.handleViewAction({
       type: userActionTypes.CURRENT_USER_ROLES_FETCH,
       userGuid,
-      orgGuid
+      entityGuid
     });
 
-    return userActions.fetchOrgUserRoles(orgGuid)
-      .then(orgRoles => userActions.receivedCurrentUserRole(orgRoles, userGuid, orgGuid));
+    if (currentViewType == SPACE_NAME){
+      return userActions.fetchSpaceUsers(entityGuid)
+        .then(entityRoles => userActions.receivedCurrentUserRole(entityRoles, userGuid, entityGuid, currentViewType));
```

Comment:
I'm fixing this problem right now in https://github.com/18F/cg-dashboard/pull/1112